File: DemoMMOUnity/Lua/application/app/ui/generateScripts/UIModules/OperationalInterfacePanelView.lua
---@class OperationalInterfacePanelView
OperationalInterfacePanelView = class("OperationalInterfacePanelView")
function OperationalInterfacePanelView:Init(view)
	self._UIView = view:GetComponent("UIView")
	self.ReturnButton = self._UIView:GetObjType("ReturnButton") or UnityEngine.UI.Button
	self.MapGuideObj = self._UIView:GetObjType("MapGuideObj") or UnityEngine.GameObject
	self.Grid = self._UIView:GetObjType("Grid") or ZJYFrameWork.Module.UICommon.AdaptiveReuseGrid
end

File: UnityMMOClient/Assets/Game/Hoftix/Script/GenerateScripts/UIModules/OperationalInterfacePanelView.cs
using ZJYFrameWork.UISerializable.UIInitView;
using UnityEngine;
using UnityEngine.UI;

namespace ZJYFrameWork.UISerializable
{
    public class OperationalInterfacePanelView:UIViewInterface
    {
        public UnityEngine.UI.Button ReturnButton=null;
		public UnityEngine.GameObject MapGuideObj=null;
		public ZJYFrameWork.Module.UICommon.AdaptiveReuseGrid Grid=null;
		


        public void Init(UIView _view)
        {
            ReturnButton=_view.GetObjType<UnityEngine.UI.Button>("ReturnButton");
			MapGuideObj=_view.GetObjType<UnityEngine.GameObject>("MapGuideObj");
			Grid=_view.GetObjType<ZJYFrameWork.Module.UICommon.AdaptiveReuseGrid>("Grid");
			
        }
    }
}